frontend/nuxt.config.ts

```diff
@@ -106,80 +51,89 @@ const openverseLocales = [
+export default defineNuxtConfig({
+  app: {
+    head: {
+      title: "Openly Licensed Images, Audio and More | Openverse",
+      meta: commonMeta,
+      link: [
+        ...favicons,
+        {
+          rel: "search",
+          type: "application/opensearchdescription+xml",
+          title: "Openverse",
+          href: "/opensearch.xml",
+        },
+        {
+          rel: "dns-prefetch",
+          href: apiUrl,
+        },
+        {
+          rel: "preconnect",
+          href: apiUrl,
+          crossorigin: "",
+        },
+      ],
     },
   },
   srcDir: "src/",
-  modern: "client",
-  server: {
-    port,
-    https: process.env.LOCAL_SSL
-      ? {
-          key: fs.readFileSync(path.resolve(__dirname, "localhost+1-key.pem")),
-          cert: fs.readFileSync(path.resolve(__dirname, "localhost+1.pem")),
-        }
-      : undefined,
-  },
-  router: {
-    middleware: "feature-flags",
-  },
-  plugins: [
-    "~/plugins/ua-parse.ts",
-    "~/plugins/focus-visible.client.ts",
-    "~/plugins/api-token.server.ts",
-    "~/plugins/polyfills.client.ts",
-    "~/plugins/sentry.ts",
-    "~/plugins/analytics.ts",
-    "~/plugins/errors.ts",
-    "~/plugins/init-stores.ts",
-  ],
-  css: ["~/assets/fonts.css", "~/styles/tailwind.css", "~/styles/accent.css"],
-  head,
-  env, // TODO: Replace with `publicRuntimeConfig`
-  privateRuntimeConfig: {
-    apiClientId: process.env.API_CLIENT_ID,
-    apiClientSecret: process.env.API_CLIENT_SECRET,
+  serverDir: "server/",
+  devServer: {
+    port: 8443,
+    host: "0.0.0.0",
+  },
+  imports: {
+    autoImport: false,
+  },
+  css: ["~/assets/fonts.css", "~/styles/accent.css"],
+  runtimeConfig: {
+    apiClientId: "",
+    apiClientSecret: "",
+    public: {
+      deploymentEnv: process.env.DEPLOYMENT_ENV ?? "local",
+      // Can be overwritten by NUXT_PUBLIC_API_URL env variable
+      apiUrl,
+      providerUpdateFrequency: 3600000,
+      savedSearchCount: 4,
+      sentry: {
+        dsn: isProdNotPlaywright
+          ? "https://b6466b74788a4a2f8a7912eea912beb7@o787041.ingest.sentry.io/5799642"
+          : "",
+        environment: isProd ? "production" : "local",
+        release: "",
+      },
+      isPlaywright,
+    },
   },
   dev: !isProd,
-  buildModules: [
-    "@nuxt/typescript-build",
-    "@nuxtjs/composition-api/module",
-    "@nuxtjs/svg-sprite",
-    "@pinia/nuxt",
-  ],
+  /**
+   * Disable debug mode to prevent excessive timing logs.
+   */
+  debug: false,
+  experimental: {
+    /**
+     * Improve router performance, see https://nuxt.com/blog/v3-10#%EF%B8%8F-build-time-route-metadata
+     */
+    scanPageMeta: true,
+  },
   modules: [
-    "portal-vue/nuxt",
+    "@pinia/nuxt",
     "@nuxtjs/i18n",
-    "@nuxtjs/proxy",
-    "@nuxtjs/redirect-module",
-    "@nuxtjs/sentry",
-    "cookie-universal-nuxt",
-    "vue-plausible",
-    "~/modules/prometheus.ts",
-    // Sitemap must be last to ensure that even routes created by other modules are added
```

Comment:
Should we keep this comment?

Reply:
The docs for nuxt sitemap don't say anything about placing the module last, so I removed it.